tests/Feature/Modules/RH/OcrServiceTest.php

```
@@ -1,40 +1,148 @@
+        ->and($data['amount_ht'])->toBe(20.42)
+        ->and($data['vat_amount'])->toBe(4.08)
+        ->and($data['merchant'])->toBe('BRICOMAN');
+});
+
+it('categorizes extracted text by merchant keywords', function (string $text, string $expected) {
+    $service = new FakeOcrParseService;
+    $service->forcedText = $text;
+
+    $data = $service->extractData('dummy.jpg');
+
+    expect($data['category'])->toBe($expected);
```

PHPStan (2.2.7)

[error] Offset 'category' does not exist on array{amount_ttc: float|null, amount_ht: float|null, vat_amount: float|null, date: string|null, merchant: string|null}.


(offsetAccess.notFound)